airflow/cli/commands/legacy_commands.py

```diff
@@ -0,0 +1,118 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from argparse import ArgumentError
+
+
+def check_value(action, value):  # pylint: disable=too-many-branches
+    # pylint: disable=too-many-statements
+    """ Checks command value and raise error if value is in removed command"""
+
+    # celery group
+    if value == "worker":
+        message = "worker command has been removed, please use `airflow celery worker`"
+        raise ArgumentError(action, message)
```

Comment:
How about
```python
def check_value(action, value):
    cmd_map = {
        "worker": "celery worker",  # old: new
        ...
    }
    try:
        msg = "{} command, has been removed, please use `airflow {}`.format(value, cmd_map[value]
        raise ArgumentError(action, msg)
    excep KeyError:
       raise ArgumentError(action, "Command {} not found".format(value))
```
We can consider making this map a global constant as it may be useful in the future.  WDYT?

Reply:
I like this!

Reply:
Yeah!

Reply:
How do you suggest to write a test for this?

Reply:
I would mock `COMMAND_MAP` with `{"old_cmd": "new_cmd"}` and then:
```python
action = MagicMock()
with self.assertrises(ArgumentError):
    check_value(action, "old_cmd")
```
plus some regex check that the error msg is valid one.

Reply:
```python
class TestCliDeprecatedCommandsValue(unittest.TestCase):
    @classmethod
    def setUpClass(cls):
        cls.parser = cli_parser.get_parser()

    def test_should_display_value(self):
        with self.assertRaises(SystemExit) as cm_exception, \
                contextlib.redirect_stderr(io.StringIO()) as temp_stderr:
            config_command.get_value(self.parser.parse_args(['worker']))

        self.assertEqual(2, cm_exception.exception.code)
        self.assertIn(
            "`airflow worker` command, has been removed, "
            "please use `airflow celery worker`, see help above.",
            temp_stderr.getvalue().strip()
        )
```
I propose to test this by testing the CLI parser. We overwrite private methods, so I wouldn't trust unit tests only.